api/goldens/aptos_api__tests__transaction_vector_test__test_module_payload.json

```diff
@@ -0,0 +1,3067 @@
+[
+  {
+    "raw_txn": {
```

Comment:
It might be good to make sure anything that contains a vector of bytes is serialized as a hex string (e.g., code)

Reply:
I wrote some code to post-process the serde_json result. It is a bit of messy. Writing a customized serializer seems to be overkilling. I guess code as bytes are OK? Since byte arrays are supported in almost any language, converting to hex string might not add much benefits. It is also less confusing to users. In our guides and tutorials, we tell people that these are bytes.

Reply:
I agree we should serialize into hex, because what we're getting is a list of bytes in decimal form, which takes up a lot more space, and is harder to differentiate.

Reply:
It should be as simple as adding `serialize_with` annotation https://serde.rs/field-attrs.html#serialize_with

Reply:
`serialize_with` for the fields in the payload structs? They are already annotated with `#[serde(with = "serde_bytes")]`. These annotation will affect BCS serialization, I believe. 

```
pub struct Script {
    #[serde(with = "serde_bytes")]
    code: Vec<u8>,
    ty_args: Vec<TypeTag>,
    args: Vec<TransactionArgument>,
}
```

I wrote code to do post JSON processing. It works. But it only helps with the `code` field. To translate the `args`, it is going to make the code really cumbersome.

I guess space is not a huge concern since we a fixed number of transactions for each type. Also, rarely people need to eyeball these to differentiate. Developers only need to follow a doc (to be written) to load these files into their test suite and do verifications.